elastic/kibana
# [Synthetics] Monitor add/edit - do not focus error via react hook form

## Summary
Resolves https://github.com/elastic/kibana/issues/144363

### Testing
1. Create your first monitor from the Synthetics app like normal
2. Navigate back to the create flow. Click the create button without adding any content. Do this for all 5 monitor types. Ensure the page does not break when not using valid values
3. Fill in appropriate values for your selected monitor type. Update the `name` field to be the name of the first monitor. YOu should get a name already exists error
4. Click create button again. Ensure that the monitor is not able to be created and the error persists
5. Choose a new name and hit create monitor again. Ensure the monitor is able to be saved successfully.
